spec/integration_helper.rb

```diff
@@ -15,14 +15,25 @@ class TestApplication < Rails::Application
   config.eager_load = false
   config.log_level = :debug
   config.action_dispatch.show_exceptions = false
-  secrets[:secret_key_base] = 'test'
-  secrets[:telegram] = {
+
+  telegram_hash = {
     bot: 'default_token',
     bots: {
       other: {token: 'other_token'},
       named: {token: 'named_token', username: 'TestBot'},
     },
   }
+
+  if Rails.version.start_with? '5.2'
```

Comment:
`Rails.gem_version >= Gem::Version.new('5.2')` should be better. This way it will support next rails versions

Reply:
Or just same as in config - using `respond_to?`.

Reply:
`Rails.application.respond_to?(:credentials)` best way